packages/text-input/package-lock.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@hashicorp/react-text-input",
```

Comment:
Wanted to check on this - noticed that in some PRs, it seems like I'm adding back a `package-lock.json` into the component's folder. Is this the correct approach? Or should these component-level `package-lock` files be deleted?

Did some quick searching and found [this suggestion that maybe I should've installed from the project root](https://stackoverflow.com/questions/65237564/npm-7-workspaces-how-to-install-new-package-in-workspace). Assuming that's the right approach, then it seems I should make sure my PRs are **not** adding component-level `package-lock.json` files. cc @BRKalow @jescalan @thiskevinwang to double-check on this 🙇

Reply:
Going to handle this in a follow-up PR once all CSS modules conversion PRs are merged (added a new task to the project board for this). I suspect the right move will be to clean up any `package-lock.json` files within component package directories, but will confirm in that upcoming PR 👍